### DISCUSSION TITLE
How to disable weak cipher suites in etcd config

### DISCUSSION THREAD

**Discussion by srbgit123** [etcd-io/etcd#15476] [Q&A] [answered]
We want to disable few weak cipher suite in etcd. Could you plz help in sharing info how to achieve that. 
I had updated the allowed ciphers by using the tag **cipher-suites:** and separated by comma. Still its showing the weak cipher along with string ones.

**Comment by jmhbnz** [comment, marked as answer]
Hey @srbgit123 - I think we can close or mark this discussion as complete with reference to https://github.com/etcd-io/etcd/discussions/15547.
